modules/table/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryReconfigurationTest.java

```diff
@@ -239,10 +270,10 @@ void testManualRebalanceIfMajorityIsLostSpecifyPartitions() throws Exception {
 
         // Should fail because majority was lost.
         List<Throwable> fixingPartErrorsBeforeReset = insertValues(table, fixingPartId, 0);
-        assertThat(fixingPartErrorsBeforeReset, Matchers.not(empty()));
+        assertThat(fixingPartErrorsBeforeReset, not(empty()));
```

Comment:
So, so, so, what do I see? 
Changes that worked before and you just changed them.
I'm not against these changes, but I'll bookmark this for future debates on this topic.

Reply:
What do you mean? That's the result of automatic refactoring by IDEA. I used method `not` in my test, particularly in line `assertThat(localStates, is(not(anEmptyMap())));`. I added static import to make that code better. Other lines have been changed as a side effect, it happens sometimes. I didn't deliberately change the code that you're pointing to, that wasn't my intention.
I see what you're trying to do. I'm not convinced that it's a comparable situation.

Reply:
I'm just having fun.